css-color/Overview.bs

```diff
@@ -1872,18 +1890,8 @@ Predefined colorspaces: ''srgb'', ''dci-p3'', and ''rec2020''.</h3>
 				<tr><th>Green chromaticity</th><td>x=0.170 y=0.797</td></tr>
 				<tr><th>Blue chromaticity</th><td>x=0.131 y=0.046</td></tr>
 				<tr><th>White chromaticity</th><td>x=0.3127 y=0.3290 (D65)</td></tr>
-				<tr><th>Transfer function</th><td>see below</td></tr>
+				<tr><th>Transfer function</th><td>1/2.4</td></tr>
```

Comment:
This is not telling the full story. Rec. 2020 specifies an OETF with an exponent of 0.45, but notes that

> in typical production practice the encoding function of image sources is adjusted so that the final
> picture has the desired look, as viewed on a reference monitor having the reference decoding
> function of Recommendation ITU-R BT.1886

BT.1886 specifies an EOTF with a gamma of 2.4.

I recommend referencing Rec.2020, instead of attempting to replicate normative text here.